esileapclient/osc/v1/node.py

```diff
@@ -48,20 +49,49 @@ def get_parser(self, prog_name):
             dest='lessee',
             required=False,
             help="Filter nodes by lessee.")
+        parser.add_argument(
+            '--property',
+            dest='properties',
+            required=False,
+            action='append',
+            help="Filter nodes by properties. Format: key>=value. \
+                Can be specified multiple times.",
+            metavar='"key>=value"')
 
         return parser
 
     def take_action(self, parsed_args):
 
         client = self.app.client_manager.lease
 
+        # Initial filters dictionary
         filters = {
             'resource_class': parsed_args.resource_class,
             'owner': parsed_args.owner,
             'lessee': parsed_args.lessee
         }
 
-        data = list(client.nodes(**filters))
+        # Add property filters if provided
+        property_filters = []
+        if parsed_args.properties:
+            for prop in parsed_args.properties:
+                key, op, value_str = utils.parse_property_filter(prop)
+                value = utils.convert_value(value_str)
+                property_filters.append((key, op, value))
```

Comment:
I'd suggest passing `parsed_args.properties` into `filter_nodes_by_properties` and doing this logic within that function; that'll make it much cleaner to add this to the lease and offer lists

Reply:
Okay, I will modify it that way.